frontend/server/src/DAO/Assignments.php

```diff
@@ -44,73 +44,83 @@ public static function getProblemset(
     /**
      * Returns each problem with the statistics of the runs submmited by the students
      *
-     * @return list<array{assignment_alias: string, problem_alias: string, variance: float}>
+     * @return list<array{assignment_alias: string, average: float|null, highScoreCount: float|null, lowScoreCount: float|null, maxPoints: float, maximum: float|null, minimum: float|null, problem_alias: string, variance: float|null}>
      */
     public static function getAssignmentsProblemsStatistics(
         int $courseId,
         int $groupId
     ): array {
         $sql = '
+        SELECT
+            bpr.assignment_alias,
+            bpr.problem_alias,
+            VARIANCE(bpr.max_user_score_for_problem) AS variance,
+            AVG(bpr.max_user_score_for_problem) AS average,
+            AVG(
+                CASE WHEN bpr.max_user_percent_for_problem > 0.6 THEN 1 ELSE 0 END
+            ) AS highScoreCount,
```

Comment:
nit:

```suggestion
            AVG(
                CASE WHEN bpr.max_user_percent_for_problem > 0.6 THEN 100 ELSE 0 END
            ) AS high_score_percentage,
```

all names in MySQL are using `snake_case` naming convention instead of `camelCase`. same in all other places where there is a `camelCase` name being assigned.

also, the word `count` confused me a bit since i thought it was going to be how many problems had a high score, but it is used as a percentage of problems that have a high score. therefore, can this be called `percentage` and be in the range [0, 100] for consistency with its intended use? Same in L64.